Library/Formula/ocproxy.rb

```diff
@@ -0,0 +1,27 @@
+class Ocproxy < Formula
+  desc "user-level SOCKS and port forwarding proxy"
+  homepage "https://github.com/cernekee/ocproxy"
+  url "https://github.com/cernekee/ocproxy/archive/v1.50.tar.gz"
+  sha256 "b061d59e0b5729d7a8950d8d4e0004a9739c092a4d5daf58a3bc61b177b24e4f"
+
+  head do
+    url "https://github.com/cernekee/ocproxy.git"
+  end
```

Comment:
Can just do `head "https://github.com/cernekee/ocproxy.git"` when it's just a URL.